spaceship/spec/test_flight/client_spec.rb

```diff
@@ -17,7 +17,6 @@ def handle_response(response)
   subject { TestFlightTestClient.new(current_team_id: 'fake-team-id') }
   let(:app_id) { 'some-app-id' }
   let(:platform) { 'ios' }
-
```

Comment:
accidental?

Reply:
Either I removed it or my ide. In any case, I can add it back. : )